partner_communication/models/communication_job.py

```diff
@@ -159,6 +166,14 @@ class CommunicationJob(models.Model):
     # printer_output_tray_id = fields.Many2one("printing.tray.output",
     #                                          "Printer Output Bin")
 
+    sms_cost = fields.Float()
+    # sms_provider_id = fields.Many2one(
+    #     "sms.provider",
+    #     "SMS Provider",
+    #     default=lambda self: self.env.ref("sms_939.large_account_id", False),
+    #     readonly=False,
+    # )
+
```

Comment:
Please remove commented code. I think providers may not be the way to work in v14, please check with David how it's supposed to work, because he migrated the module `sms_939` and knows how the SMS is supposed to be sent. Make sure that the code in this file is not dependent on a Swiss specific provider but that it can be set by the user or by a setting.